docs/Console/README.md

```diff
@@ -130,24 +129,24 @@ this mode, the solution must be manually built. To do so, please follow these
 steps:
 
 1. Clone the project: `git clone https://github.com/Microsoft/dotnet-apiport`
-2. Compile as normal, either using `msbuild` or Visual Studio
-3. If in Visual Studio, set the project `ApiPort.Offline` as the startup project, or if from command line, goto `bin\[Configuration]\ApiPort.Offline` and run `ApiPort.exe` from this directory
+2. Compile using the `build.cmd` or `build.sh` script as normal
+3. If in Visual Studio, set the project `ApiPort.Offline` as the startup project, or if from command line, goto `bin\[Configuration]\ApiPort.Offline\net46` and run `ApiPort.exe` from this directory.
+
+There is a .NET Core build of the offline mode, but HTML reports will not be generated when running on .NET Core. Other than that, the functionality is expected to be the same.
 
 ### Proxies
 
 If you have issues with proxies, please add your proxy information to the catalog script:
 
-1. Edit [`.\build\Get-CatalogFile.ps1` at line 24](https://github.com/Microsoft/dotnet-apiport/blob/master/build/Get-CatalogFile.ps1#L24)
-2. Replace Line 24 with:
+1. Edit [`.\init.ps1` at line 19](../../init.ps1#L19)
```

Comment:
Would it be better to list the line to be replaced, instead of line number here? That way future changes in `init.ps1` doesn't require changes in this doc.

Reply:
Good call